File: test-suite/src/test/java/io/micronaut/docs/client/filter/BasicAuthClient.java
package io.micronaut.docs.client.filter;

//tag::class[]
import io.micronaut.http.annotation.Get;
import io.micronaut.http.client.annotation.Client;

@BasicAuth // <1>
@Client("/message")
public interface BasicAuthClient {

    @Get
    String getMessage();
}
//end::class[]

File: test-suite-python/src/test/python/micronaut/docs/client/filter/BasicAuthClient.py
from abc import abstractmethod

# tag::class[]
from micronaut.http.annotation import Get
from micronaut.http.client.annotation import Client

from .BasicAuth import BasicAuth


@BasicAuth  # <1>
@Client("/message")
class BasicAuthClient:

    @Get
    @abstractmethod
    def getMessage(self) -> str:
        pass
# end::class[]
